src/dlgProfilePreferences.cpp

```diff
@@ -615,7 +624,7 @@ void dlgProfilePreferences::setCommandBgColor()
 
 void dlgProfilePreferences::setFontSize()
 {
-    mFontSize = fontSize->currentIndex();
+    mFontSize = fontSize->currentIndex() + 1;
```

Comment:
💡 I think somewhere back in the depths of time there was a bug report about the size being set not the value entered - and I think this would have been the cause - let me see, yeah #781 (Launchpad Bug Details: [#LP1421505](https://bugs.launchpad.net/bugs/1421505) lights4drowning - 2015-02-13 03:17:52 +0000) - well done in spotting this...! 🎉